manipulation/packages/frida_motion_planning/examples/add_collision_object.py

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env python3
+
+import rclpy
+from rclpy.node import Node
+from frida_interfaces.srv import AddCollisionObject
+from geometry_msgs.msg import Quaternion
+from std_msgs.msg import Header
+
+"""
+Usage
+- ros2 run frida_motion_planning add_collision_object.py --ros-args -p type:="box" -p position:="[0.5, 0.0, 0.5]" -p dimensions:="[0.2, 0.2, 0.05]"
+- ros2 run frida_motion_planning add_collision_object.py --ros-args -p type:="cylinder" -p position:="[0.2, 0.0, 0.5]" -p quat_xyzw:="[0.0, 0.0, 0.0, 1.0]" -p dimensions:="[0.1, 0.4]"
+- ros2 run frida_motion_planning add_collision_object.py --ros-args -p type:="sphere -p position:="[0.2, 0.0, 0.5]" -p dimensions:="[0.15]"
+- ros2 run frida_motion_planning add_collision_object.py --ros-args -p type:="mesh" -p position:="[0.2, 0.0, 0.5]" -p quat_xyzw:="[0.0, 0.0, 0.0, 1.0]" -p mesh_file:="package://frida_motion_planning/meshes/box.stl"
+"""
+
+
+def main():
+    rclpy.init()
+
+    node = Node("ex_collision_primitive")
+    node.declare_parameter("type", "box")
+    node.declare_parameter("position", [0.5, 0.0, 0.5])
+    node.declare_parameter("quat_xyzw", [0.0, 0.0, 0.0, 1.0])
+    node.declare_parameter("dimensions", [0.1, 0.1, 0.1])
+    node.declare_parameter("mesh_file", "")
+
+    add_collision_object_client = node.create_client(
+        AddCollisionObject, "/manipulation/add_collision_object"
```

Comment:
can we add these to constants instead of raw strings here
```suggestion
        AddCollisionObject, CONSTANT
```

Reply:
preferably use frida constants